app/src/main/res/layout/biometric_prompt_dialog_content.xml

```diff
@@ -26,15 +26,15 @@
         android:gravity="center">
 
         <ImageView
-            android:id="@+id/fingerprint_icon"
+            android:id="@+id/fingerprintIcon"
```

Comment:
nit: I would keep these as snake_case to be consistent. Another perk that I've noticed is that it's easier to tell in the Kotlin files that the variables are generated ones from the xml